Library/Formula/scrypt.rb

```diff
@@ -13,7 +14,13 @@ class Scrypt < Formula
 
   depends_on "openssl"
 
+  head do
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+  end
+
   def install
+    system "autoreconf", "-i" if build.head?
```

Comment:
Prefer `system "autoreconf", "-fvi" if build.head?` for consistency. Trying to unify these lines a little more.